packages/optimizer/lib/transformers/AddBlurryImagePlaceholders.js

```diff
@@ -48,11 +52,28 @@ function escaper(match) {
  * * `blurredPlaceholders`: Enables blurry image placeholder generation. Default is `false`.
  * * `imageBasePath`: specifies a base path used to resolve an image during build.
  * * `maxBlurredPlaceholders`: Specifies the max number of blurred images. Defaults to 5.
+ * * `blurredPlaceholdersCacheSize`: Specifies the max number of blurred images to be cached
+ *   to avoid expensive recalculation. Set to 0 if all placeholders should be cached. Defaults
+ *   to 30.
  *
  * Important: blurry image placeholder computation is expensive. Make sure to
  * only use it for static or cached pages.
  */
 class AddBlurryImagePlaceholders {
+  constructor(config) {
```

Comment:
nit: The next two lines could be written as:

```
constructor({blurredPlaceholdersCacheSize = DEFAULT_CACHED_PLACEHOLDERS} = {}) {
```

Reply:
that's too much magic for me